spec/javascript/components/molecules/LeaseUpSidebarButtons.test.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
-import renderer from 'react-test-renderer'
+import { shallow } from 'enzyme'
 import LeaseUpSidebarButtons from 'components/molecules/LeaseUpSidebarButtons'
 
 describe('LeaseUpSidebarButtons', () => {
   test('should render correctly with mobile styling', () => {
-    const wrapper = renderer.create(
+    const wrapper = shallow(
```

Comment:
Instead of using `react-test-renderer`'s `render.create` to make wrappers, use [shallow](https://enzymejs.github.io/enzyme/docs/api/shallow.html) instead. 

The react-test-renderer create method deeply renders a component, meaning it renders the component's elements and all of its children's components elements. This makes the snapshots huge and hard to read, and means our unit tests for wrapper components are actually testing every child component they contain.

Shallow only renders one level deep, so say you're writing a test for MyComponent:

```
const MyChildComponent = (someProp) => (
<div>
  <div className="div3">
    <div className="div4" />
  </div>
  <div className="div5">
    <div className="div6" />
  </div>
</div>
)

const MyComponent = () => (
<div>
  <div className="div1">
    <div className="div2" />
  </div>
  <MyChildComponent someProp={true} />
  <MyChildComponent someProp={false} />
</div>
)
```

The shallow snapshot would look like:
```
<div>
  <div className="div1">
    <div className="div2" />
  </div>
  <MyChildComponent someProp={true} />
  <MyChildComponent someProp={false} />
</div>
```
and the renderer.create snapshot would look like:
```
<div>
  <div className="div1">
    <div className="div2" />
  </div>
  <div>
  <div className="div3">
    <div className="div4" />
  </div>
  <div className="div5">
    <div className="div6" />
  </div>
  </div>
  <div>
    <div className="div3">
      <div className="div4" />
    </div>
    <div className="div5">
      <div className="div6" />
    </div>
  </div>
</div>
```